src/chess.py

```diff
@@ -1,9 +1,108 @@
 """A game of chess."""
+from src.board import Board
+from src.pieces import Color, Piece
+from src.square import Square
+
+
+Score = tuple[int, int]
+Checkmate = bool
 
 
 class Chess:
     """A chess game."""
 
     def __init__(self):
         """Start a chess game."""
-        ...
+        self._board = Board()
+
+    def _take_turns(self, color: Color) -> tuple[Score, Checkmate]:
+        """A player's turn, where he selects a piece to move and a target square to move to.
+        Args:
+            color: the color of the player
+        Returns:
+            a tuple of the current score and whether the game has ended with checkmate.
+        """
```

Comment:
@palasso if Google docstrings support not having black lines in between, I like this.

Reply:
I will fix it.

Reply:
I wasn't being ironic, I like this better. But not sure whether @palas... erm Google will agree. :P